src/shapes.ts

```diff
@@ -451,21 +451,7 @@ export interface ExternalWindow extends BrowserWindowElectron {
     uuid: string;
 }
 
-export interface RawNativeWindowInfo {
-    alwaysOnTop: boolean;
-    bounds: Bounds;
-    className: string;
-    dpi: number;
-    focused: boolean;
-    id: string;
-    maximized: boolean;
-    minimized: boolean;
-    process: ProcessInfo;
-    title: string;
-    visible: boolean;
-}
-
-export interface NativeWindowInfo extends RawNativeWindowInfo {
+export interface NativeWindowInfo extends NativeWindowInfo {
```

Comment:
Seems odd to me to use the same name as the import from electron here. Maybe rename the electron import to something like `NativeWindowInfoElectron`(as we do for the `BrowserWindow` import).

This is a nitpick though and shouldn't really affect anything beyond aesthetics, so I'll go ahead and approve. But seems like something that might cause some small amount of future confusion.

Reply:
will change now